### ISSUE TITLE
Infix function `**` mistaken for a wildcard import

### ISSUE THREAD

**Issue by binkley** [pinterest/ktlint#799]
## Expected Behavior

Ktlint does not complain about importing an infix function named `**` as being a wildcard import.  This infix function acts as a "pseudo-operator" for exponentiation

## Observed Behavior

For the import:
```
import hm.binkley.math.`**`
```
Ktlint mistakenly complains this is a wildcard import

## Steps to Reproduce
See this commit: https://github.com/binkley/kotlin-rational/commit/2c610d6122c0b7cb2ae970d171b92e2c30fdb269

I am grateful for the comment syntax that Ktlint recognizes, so I could ask it to ignore the import in question

## Your Environment
* Version of ktlint used: 1.5.2 (latest as per Maven Central)  Please note: the commit link above was using Ktlint 1.4.2 in the `pom.xml`.  A following commit is using version `1.5.2`, and has the same issue.  I linked to the earlier commit as it was more clear in the code that Ktlint complained about
* Name and version (or code for custom task) of integration used (Gradle plugin, Maven plugin, command line, custom Gradle task): Maven plugin
* Version of Gradle used (if applicable): n/a
* Operating System and version:
```
$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=20.04
DISTRIB_CODENAME=focal
DISTRIB_DESCRIPTION="Ubuntu 20.04 LTS"
```
* Link to your project (if it's a public repository): https://github.com/binkley/kotlin-rational
